docs/4.0/architecture.md

```diff
@@ -108,7 +108,7 @@ node:
     for the two-step authentication. The password + 2nd factor are submitted to a proxy via HTTPS, therefore it is critical for a secure configuration of Teleport to install a proper HTTPS certificate on a proxy.
 
 !!! warning "Warning":
-	Do not use a self-signed certificate in production!
+	Do not use a self-signed SSL/HTTPS certificates when creating production! 
```

Comment:
Let's stick to HTTPS certificates, SSL is kind of an old, deprecated term

Reply:
Since this is a high level message, I've kept it as SSL/HTTPS for extra clarification based on current traffic for those search terms.